setup1-verifier/src/main.rs

```diff
@@ -16,57 +18,69 @@ fn development() -> Environment {
     .into()
 }
 
-#[inline]
 fn inner() -> Environment {
     Production::from(Parameters::AleoInner).into()
 }
 
-#[inline]
 fn outer() -> Environment {
     Production::from(Parameters::AleoOuter).into()
 }
 
-#[inline]
 fn universal() -> Environment {
     Production::from(Parameters::AleoUniversal).into()
 }
 
+#[derive(Debug, StructOpt)]
+#[structopt(name = "Aleo setup verifier")]
+struct Options {
+    #[structopt(long, help = "Path to a file containing verifier view key")]
+    view_key: PathBuf,
+    #[structopt(long, help = "Coordinator api url, for example http://localhost:9000")]
+    api_url: Url,
+}
+
+async fn request_coordinator_public_settings(coordinator_url: &Url) -> anyhow::Result<PublicSettings> {
```

Comment:
could this code be shared with the contributor?

Reply:
I don't think there's a point in sharing such a small function at the moment. But maybe in the future it'll be more obvious how can we share the networking parts between verifier and contributor.